runtime/JavaScript/src/antlr4/Recognizer.d.ts

```diff
@@ -6,4 +6,7 @@ export declare class Recognizer<TSymbol> {
 
     removeErrorListeners(): void;
     addErrorListener(listener: ErrorListener<TSymbol>): void;
+    getErrorListenerDispatch(): ErrorListener<TSymbol>;
```

Comment:
can you rename to getErrorListener() and add to js ?

Reply:
... and I think the `getErrorListenerDispatch` method should be marked as depricated in js.